assets/src/components/selectControl/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Muriel-styled Select dropdown.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { SelectControl as BaseComponent, withFocusOutside } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import murielClassnames from '../../shared/js/muriel-classnames';
+import './style.scss';
+
+const SelectControl = withFocusOutside(
+
+	class extends Component {
+
+		/**
+		 * Constructor.
+		 */
+		constructor( props ) {
+			super( props  );
+			this.state = {
+				isFocused: false
+			};
+		};
+
+		/**
+		 * A `withFocusOutside` HOC's default handler.
+		 */
+		handleFocusOutside() {
+			this.setState( { isFocused: false } );
+		};
+
+		/**
+		 * Handles component onChange; also executes the props' onChange handler (the parent's original handler).
+		 */
+		handleOnChange = ( onChange, value ) => {
+			onChange( value );
+			this.setState( { isFocused: false } );
+		};
+
+		/**
+		 * Handle component onClick.
+		 */
+		handleOnClick = () => {
+			this.setState( { isFocused: true } );
+		};
+
+		/**
+		 * Get component's className describing its state.
+		 */
+		getClassName = ( disabled, isEmpty, isActive ) => {
+			let className = "with-value";
+			if ( disabled ) {
+				className = "disabled";
+			} else if ( isActive ) {
+				className = "active";
+			} else if ( isEmpty ) {
+				className = "empty";
+			}
+
+			return className;
+		};
+
+		/**
+		 * Render.
+		 */
+		render() {
+			const { isFocused } = this.state;
+			const { value, disabled, onChange } = this.props;
+			const isEmpty = ! value;
+			const isActive = isFocused && ! disabled;
+
+			return (
+				<BaseComponent
+					className={ murielClassnames( "muriel-select", this.getClassName( disabled, isEmpty, isActive ) ) }
```

Comment:
`className` should be handled using the pattern [here](https://github.com/Automattic/newspack-plugin/pull/61/files#diff-8a670ed255f49ef836328cad94e50247R24). Otherwise, setting a `className`  overrides all of the classnames.

e.g. try this:
```
<SelectControl
	className="testing"
	label="Select with value"
	value={ selectValue1 }
	options={ [
		{ value: '1st', label: 'First' },
		{ value: '2nd', label: 'Second' },
		{ value: '3rd', label: 'Third' },
	] }
	value={ selectValue1 }
	onChange={ value => this.setState( { selectValue1: value } ) }
/>
```

You can see that our classes get removed and the only class is 'testing':

<img width="470" alt="Screen Shot 2019-05-14 at 12 00 35 PM" src="https://user-images.githubusercontent.com/7317227/57724604-08da0080-7640-11e9-9d6a-7359f97a0b68.png">

Reply:
Great thinking! Thank you! Updated now.

I also love how the spread operator is used to handle the `...otherProps` in that example, so I added this change as well.